internal/action/otp.go

```diff
@@ -65,25 +66,52 @@ func (s *Action) otp(ctx context.Context, name, qrf string, clip, pw, recurse bo
 		skip = true
 	} else { // if not then we want to print a progress bar with the expiry time
 		out.Printf(ctx, "%s", token)
-		out.Warningf(ctx, "This OTP password still lasts for:", nil)
+		out.Warningf(ctx, "([q] to stop. -o flag to avoid.) This OTP password still lasts for:", nil)
 		bar := termio.NewProgressBar(int64(secondsLeft))
 		bar.Hidden = ctxutil.IsHidden(ctx)
 		if bar.Hidden {
 			skip = true
 		} else {
+			cancel := make(chan bool)
 			bar.Set(0)
 			go func() {
 				ticker := time.NewTicker(1 * time.Second)
 				defer ticker.Stop()
+				cancelled := false
 				for tt := range ticker.C {
-					if tt.After(expiresAt) {
+					select {
+					case <-cancel:
+						cancelled = true
+					default:
+						// we don't want to block if not cancelled
+					}
+					if tt.After(expiresAt) || cancelled {
 						bar.Done()
 						done <- true
 						return
 					}
 					bar.Inc()
 				}
 			}()
+			go func() {
+				tty, err := tty.Open()
```

Comment:
Could you remind me why we need the tty lib here?
Can we not just do a regular [read](https://github.com/gopasspw/gopass/blob/master/pkg/termio/ask.go#L43)?